src/main/java/seedu/address/storage/JsonAdaptedPatient.java

```diff
@@ -35,11 +34,16 @@ class JsonAdaptedPatient {
      */
     @JsonCreator
     public JsonAdaptedPatient(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
-                              @JsonProperty("email") String email, @JsonProperty("address") String address,
+                              @JsonProperty("email") String email, @JsonProperty("gender") String gender,
+                              @JsonProperty("icNumber") String icNumber, @JsonProperty("birthday") String birthday,
+                              @JsonProperty("address") String address,
                               @JsonProperty("tags") List<JsonAdaptedTag> tags) {
         this.name = name;
         this.phone = phone;
         this.email = email;
+        this.gender = gender;
```

Comment:
Hmm it appears that the patient's department is not stored here. I feel like it might be needed though, in the case where for whatever reason the user has to relaunch the app (e.g. in the middle of a shift) and finds that all the data about the patient's department is gone (then they wouldn't know where each patient is now).

Reply:
I was thinking we can overload the constructor and have two constructors. One can be for the reading of addressbook from file (need to store department info) and we can have the constructor take in a department string as an argument. The other is for creating new patient objects (like for the add command), then we don't need the department param.

Reply:
Also we need to eventually find a way to store the records into the hard disk as well 😔

Reply:
You are right, i think i will create a github task for the next iteration. Definitely will have to focus on storage next iteration!!